core/src/main/scala/io/finch/request/package.scala

```diff
@@ -198,14 +202,15 @@ package object request extends LowPriorityImplicits {
   object RequiredParam {
 
     /**
-     * Creates a ''RequestReader'' that reads a required string ''param''
-     * from the request or raises an exception when the param is missing or empty.
+     * Creates a [[RequestReader]] that reads a required string ''param'' from the request or raises a [[NotPresent]]
```

Comment:
It definitely works for 2.11 (just checked it). So, we're Ok with the web-site. Although, it doesn't work inside Intellij, so I will fix the docs to make them work everywhere. Thanks for pointing this!